src/RelatedViewWidget.js

```diff
@@ -59,25 +63,37 @@ define('Sage/Platform/Mobile/RelatedViewWidget', [
         selectMoreDataText2: 'see ${0} more ... ',
         navToListText:'see list',
         loadingText: 'loading ... ',
-        refreshViewText: 'refresh',
         itemOfCountText: ' ${0} of ${1}',
-        totalCountText: ' (${0})',
+        totalCountText: ' ${0}',
+        deleteText: 'Delete',
+        editText: 'Edit',
+        addText: 'text',
+        viewDetailText: 'view detail',
+        vliewListText:'view list',
+        refreshText: 'refresh',
+
+        owner: null,
         parentProperty: '$key',
         parentEntry: null,
         relatedProperty: '$key',
+        relatedItemKeyProperty:'$key', 
         relatedEntry: null,
         itemsNode: null,
         loadingNode: null,
         id: 'related-view',
+        iconDrillToDetail: 'content/images/icons/drilldown_24.png',
+        icon: 'content/images/icons/ContactProfile_48x48.png',
+        itemIcon: 'content/images/icons/ContactProfile_48x48.png',
         titleText: 'Related View',
         detailViewId: null,
+        editViewId: null,
         listViewId: null,
         listViewWhere: null,
-        enabled: false,
         parentCollection: false,
         parentCollectionProperty: null,
+        parentResourceKind: null,
```

Comment:
Having all these SData specific properties feels kinda wrong here. Should we split it into a base mixin and use sdata specific mixins on top of that?